app/views/kaminari/_page.html.slim

```diff
@@ -6,6 +6,10 @@
     total_pages  : total number of pages
     per_page     : number of items to fetch per page
     remote       : data-remote
-li.page-item class="page#{' current' if page.current?}"
-  == link_to_unless page.current?, page, url, {remote: remote, rel: page.rel, class: 'page-link'}
+li.page-item class="page#{' active current' if page.current?}"
```

Comment:
👍  in Rails 6.1 dann mit `class_names` ❤️  https://www.rubydoc.info/docs/rails/ActionView%2FHelpers%2FTagHelper:class_names